src/test/resources/org/jpeek/metrics/SkippedEnum.java

```diff
@@ -0,0 +1,4 @@
+public enum SkippedEnum {
```

Comment:
Why this is needed?

Reply:
Codecov shows not covered lines
 && !ctClass.isEnum()
&& !ctClass.isAnnotation()
so it was a try to grow up coverage.
But I don't see any enhancement there(

Reply:
@magdel this looks weird :) please, remove these files

Reply:
ok